### DISCUSSION TITLE
Break Time

### DISCUSSION THREAD

**Discussion by benroscoe** [blakeblackshear/frigate#915] [Ask A Question]
So far I am loving frigate. The detection is significantly better than built in Blue Iris and being able to use Coral has really made detect work again compared to other Tensor Flow stuff on  the CPU.

One feature I'm trying to determine if it exists yet is what Blue Iris calls "break time". I have some cameras were I don't want to spam detection events when people are present. So after a first detection I wouldn't want another detection to occur until a full XXX seconds have passed. 

So for example, detect a person coming into a room. They move around for 20 seconds and stand still for a little bit and motion detection stops. Then a few seconds later they start moving again for 20 seconds and then stop. Then move again, leaving the room. A detection should occur again until say 60 seconds after they leave the room.

I know how you define the recorded clip could be tricky. I would say it should record the first YYY seconds, as defined by the max_seconds setting, then end the clip. 

I haven't seen anything like that so far. Have I missed it?

![image](https://user-images.githubusercontent.com/2783568/111720671-64710880-882c-11eb-9384-c8d1ff3f3df1.png)

**Comment by blakeblackshear** [comment]
I think most people handle this within homeassistant or nodered automations.

**Reply by benroscoe** [reply]
Interesting. So just ignore updates from Frigate for however long you need to? I suppose that works, just end the alert on the first thing and then ignore it for awhile.
I guess I'm curious, what ends an event? If motion stops do you stop sending frames into the coral for analysis? Or does the coral always keep analyzing the frames? 
I ask because it seemed like I could stand there in a way that in theory the model would have know it was a person. But it seemed like because I was still the detection stopped. Just wanted to confirm that was the intended functionality.

**Reply by blakeblackshear** [reply]
Recent versions stop when motion stops to avoid constantly running detection on things like parked cars. A new event will trigger if the car was to start moving again later. The event ends when enough frames without motion go by or when the tracked object leaves the screen. Homeassistant has builtin controls for throttling in automations.

**Comment by TurboTronix** [comment]
Yeah i did that via hassio in my notification automation.
